lib/maps/fanv2.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (_, Service, Characteristic) => ({
+  class: ['thermostat', 'heatpump'],
+  service: Service.Fanv2,
```

Comment:
Is there an upside of using `Service.Fanv2` over `Service.Fan`? If so, I might consider using it for device class `fan` as well.

Reply:
I'm not sure, I need to investigate.

For now it's juste a draft, the idea is to update everywhere it's possible 😊

Reply:
Fair enough 😅

Reply:
From my understanding, Service.Fan is on-off switch button with optional fan speed slider. 

Service.FanV2 is a single slider handling both fan-speed and on-off (0), with 2 distinct characteristics to handle home screen's quick action and/or when device has 2 separate capabilities (I guess).

Reply:
Yeah I was looking at the [class definition](https://github.com/homebridge/HAP-NodeJS/blob/ee41309fd9eac383cdcace39f4f6f6a3d54396f3/src/lib/definitions/ServiceDefinitions.ts#L515) and `Fanv2` also supports swing mode, which Homey now also has a capability for, so I might rewrite the existing fan to use the v2 service instead.

Reply:
Some time ago, some "fan" Homey apps only have an on-off button on Homey, so maybe keeping `Service.Fan` could be useful in some cases, like when dim / fan_speed is missing, otherwise it could be migrated to `Service.FanV2`.

Reply:
Good point, something to think about.

Reply:
Should be easy to implement: `fan-onoff.js` + `forbidden : [ 'dim', 'fan_speed' ]`

Reply:
https://github.com/robertklep/name.klep.homekitty/pull/111/commits/bf0a11511a8c98d96feb9d65cacf9f701a5a155b

Reply:
Unfortunately [Homey swing mode](https://github.com/athombv/node-homey-lib/blob/66ba69571be3c3d374b7472dc37b3ee2e39e2a2b/assets/capability/capabilities/swing_mode.json#L16) doesn't fit [HomeKit swing mode](https://github.com/homebridge/HAP-NodeJS/blob/ee41309fd9eac383cdcace39f4f6f6a3d54396f3/src/lib/definitions/CharacteristicDefinitions.ts#L4028)